### PR TITLE
#54: updated Forest taxonomies and form

### PR DESCRIPTION
**Pull request by Yash621**
- Changed Asset type "Forest" label to "Stand"
- updated Forest vegetation type taxonomy to: 'Tropical High Forest', 'Woodland', 'Bush',
- changed Forest type select options to be either Natural stand OR Plantation, not both.
- implemented selecting Natural stand to only show the natural forest stand details
- implemented selecting Plantation to only show the plantation stand details
- Created terms for the Purpose taxonomy: 
   - For Plantation, available terms are: "Poles", "Timber", "Fuel", or "Processing"
   - For Natural stand, available terms are: 'conservation' or 'restoration'

Demo: 

https://github.com/National-Forestry-Authority/forests/assets/58719389/89a9f5e0-6ff5-4625-9793-0c6dcacb8462

### CODE REVIEW

File: web/modules/custom/farm_nfa/farm_nfa.install [hunk 1]
```diff
@@ -273,3 +274,69 @@ function farm_nfa_update_9019(&$sandbox) {
...
+        'name' => $term,
+        'vid' => $current_vocab,
+      ));
+      $term->save();
+    }
+  }
+}
+
+/**
+ * save the taxonomy terms for forest purpose taxonomy type
```
Comment (peacog) [inline web/modules/custom/farm_nfa/farm_nfa.install:308]: Coding standards - Comments should start with a capital letter and end with a full-stop.

File: web/modules/custom/farm_nfa/farm_nfa.install [hunk 2]
```diff
@@ -273,3 +274,69 @@ function farm_nfa_update_9019(&$sandbox) {
...
+    'Conservation' => [
+      'forest_type' => 'natural',
+    ],
+    'Restoration' => [
+      'forest_type' => 'natural',
+    ]
+  ];
+  foreach ($forest_purpose_terms as $term => $term_info) {
+    // Term names to be added.
+    $current_term = Term::create(array(
```
Comment (peacog) [inline web/modules/custom/farm_nfa/farm_nfa.install:334]: Use short array syntax

File: web/modules/custom/farm_nfa/farm_nfa.install [hunk 3]
```diff
@@ -273,3 +274,69 @@ function farm_nfa_update_9019(&$sandbox) {
...
+      'parent' => array(),
```
Comment (peacog) [inline web/modules/custom/farm_nfa/farm_nfa.install:335]: Remove 'parent'

File: web/modules/custom/farm_nfa/farm_nfa.install [hunk 4]
```diff
@@ -273,3 +274,69 @@ function farm_nfa_update_9019(&$sandbox) {
...
+      'name' => $term,
+      'vid' => $vocab,
+      'forest_type' => $term_info['forest_type'],
+    ));
+    $current_term->save();
+  }
+}
```
Comment (peacog) [inline web/modules/custom/farm_nfa/farm_nfa.install:342]: As per coding standards, files should always end with a single empty line: https://www.drupal.org/docs/develop/standards/php/php-coding-standards#indenting

File: web/modules/custom/farm_nfa/farm_nfa.install [hunk 5]
```diff
@@ -273,3 +274,69 @@ function farm_nfa_update_9019(&$sandbox) {
...
+ * save the taxonomy terms for forest_vegetation and forest_type taxonomy types.
+ */
+function farm_nfa_update_9020(&$sandbox) {
+  $vocab_terms = [
+    'forest_vegetation' => [
+      'Tropical High Forest',
+      'Woodland',
+      'Bush',
+    ],
+    'forest_type' => [
```
Comment (peacog) [inline web/modules/custom/farm_nfa/farm_nfa.install:288]: Do we need forest_type in this hook?

Comment (Yash621) [reply web/modules/custom/farm_nfa/farm_nfa.install]: yea @peacog correct, i forgot we changed our approach and started setting the forest_type from the farm_nfa_forest.module manually . have updated this accordingly, thanks !

**Pull request merged**